### ISSUE TITLE
3 gateways stopped working

### ISSUE THREAD

**Issue by anonymous-one** [fronzbot/blinkpy#897]
I apologize for posting this here but to be honest it seams like the most appropriate place to do so.

I just had 3 gateways stop working out the blue. Not sure if amazon / blink pushed a new firmware update but here goes:

On reboot (we had a power outage), they refused to connect to my wireless network (AP). I checked my logs on the APs there is _NO_ auth request from their mac addresses.

When I tried to do the reset button + change wifi via the app, the app finds my AP no problem but when the gateway goes to connect it ends up on "red light". Again, nothing in the logs on the access points. Its as though the gateways do not even attempt to connect to them. Of course I tried multiple times and a few low hanging fruit (eg: channel change etc). Nothing. They all end up with solid "red light" fairly quickly. The app says "probably wrong password" but thats just a blanket statement as they authenticate with the APs let alone connect to the internet, so the app has no way of knowing really.

I will add, my 2.4ghz network is very vanilla. Nothing special. On top of that I even tried connecting to a mobile hotspot from an android device. Same issue.

I'll finish with this:

I am 100% sure this is not my network. I am fairly well versed in general networking as well as wifi, all of my APs are OpenWRT based and I have about a decade of use using OpenWRT etc. 

Last but not lease I'll add that I had 3 gateways because in the past (before blinkpy) I used IFTTT so thats how I seperated my network into exterior / interior etc.  One more time, 100% sure its not my network :)

Anyone else experience the same?

...a real shame. I have about 10 of these cameras!

**Comment by anonymous-one** [comment]
Ok!

Small update, I know I just posted but I am in the middle of doing this as we speak:

I was able to get one to come back to life by doing the following:

Hold down the reset button, plug in power. Red light will start flashing. I am guessing this rolls back the firmware to the original shipped (?).

At this point I was able to get it to connect to my wifi.

The app updated the firmware at the end of the add to account.

I tried unplug + plug and it reconnected.
